From fb1f0b0036421191ad5d7f581f812785768416e5 Mon Sep 17 00:00:00 2001
From: Andreas Sturmlechner <asturm@gentoo.org>
Date: Wed, 9 Jul 2025 20:51:24 +0200
Subject: [PATCH] org.kde.konqueror.desktop: Set Type to "Application", fix
 Categories

Fixes downstream QA notices:

```
 * org.kde.konqueror.desktop: error: key "Categories" is present in group "Desktop Entry", but the type is "Service" while this key is only valid for type "Application"
 * org.kde.konqueror.desktop: error: key "Exec" is present in group "Desktop Entry", but the type is "Service" while this key is only valid for type "Application"
 * org.kde.konqueror.desktop: error: key "StartupNotify" is present in group "Desktop Entry", but the type is "Service" while this key is only valid for type "Application"
 * org.kde.konqueror.desktop: error: value "Qt;KDE;Internet;Network;" for key "Categories" in group "Desktop Entry" contains an unregistered value "Internet"; values extending the format should start with "X-"
```

Signed-off-by: Andreas Sturmlechner <asturm@gentoo.org>
---
 org.kde.konqueror.desktop | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/org.kde.konqueror.desktop b/org.kde.konqueror.desktop
index e3ad71e369..bae4f6910c 100644
--- a/org.kde.konqueror.desktop
+++ b/org.kde.konqueror.desktop
@@ -1,5 +1,5 @@
 [Desktop Entry]
-Type=Service
+Type=Application
 NoDisplay=true
 Name=Konqueror
 Name[af]=Konqueror
@@ -229,7 +229,7 @@ Comment[xh]=Umphathi Wefayile ye KDE & Umkhangeli zincwadi we Web
 Comment[x-test]=xxKDE File Manager & Web Browserxx
 Comment[zh_CN]=KDE 文件管理器和 Web 浏览器
 Comment[zh_TW]=KDE 檔案管理程式和網頁瀏覽器
-Categories=Qt;KDE;Internet;Network;
+Categories=Qt;KDE;Network;WebBrowser;
 Icon=konqueror
 X-DBUS-StartupType=multi
 StartupNotify=true
-- 
2.50.1

